python/point_cloud2.py

```diff
@@ -357,6 +357,120 @@ def make_radar_point_cloud(header: Header, points: Iterable, is_dense: bool = Tr
     return create_cloud(header, fields, points, is_dense)
 
 
+def create_entities_cloud(
```

Comment:
@ClementBotty I don't get why this function is even needed. Just a single call. Besides, `:param is_dense: True if there are no invalid points` us described but not present

Reply:
I removed `create entities_cloud` and now I don't use 'is_dense' anymore.